src/components/sale/MemberPhoneModal.tsx

```diff
@@ -0,0 +1,231 @@
+import { CheckOutlined, CloseOutlined, PlusOutlined, StopOutlined } from '@ant-design/icons'
+import { gql, useMutation } from '@apollo/client'
+import { Button, Form, Input, List } from 'antd'
+import { useForm } from 'antd/lib/form/Form'
+import { CommonTitleMixin } from 'lodestar-app-element/src/components/common'
+import React, { useState } from 'react'
+import { useIntl } from 'react-intl'
+import styled from 'styled-components'
+import { commonMessages, salesMessages } from '../../helpers/translation'
+import AdminModal from '../admin/AdminModal'
+
+const StyledModalTitle = styled.div`
+  ${CommonTitleMixin}
+`
+
+const StyledDelPhone = styled.p`
+  color: var(--gray);
+`
+
+const StyledNewPhoneInput = styled.div`
+  width: 80%;
+  display: flex;
+  align-items: center;
+  gap: 12px;
+`
+const StyledNewPhoneButton = styled(Button)`
+  fontsize: 16px;
+  fontweight: 500;
+`
+
+type FieldProps = {
+  phone: string
+}
+
+const MemberPhoneModal: React.FC<{
+  onCancel: () => void
+  visible: boolean
+  phones: {
+    phoneNumber: string
+    isValid: boolean
+  }[]
+  memberId: string
+  onLeadRefetch: () => Promise<void>
+}> = ({ onCancel, onLeadRefetch, visible, phones, memberId }) => {
+  const { formatMessage } = useIntl()
+  const [form] = useForm<FieldProps>()
+  const [isSubmitting, setIsSubmitting] = useState(false)
+  const [updatePhoneNumber, setUpdatePhoneNumber] = useState<{ phoneNumber: string; isValid: boolean }[]>([])
+
+  const [insertMemberPhone] = useMutation(
+    gql`
+      mutation InsertMemberPhone($phones: [member_phone_insert_input!]!) {
+        insert_member_phone(objects: $phones) {
+          affected_rows
+        }
+      }
+    `,
+  )
+  const [updateMemberPhone] = useMutation(gql`
+    mutation UpdateMemberPhone($memberId: String!, $phoneNumber: String!, $isValid: Boolean!) {
+      update_member_phone(
+        where: { member_id: { _eq: $memberId }, phone: { _eq: $phoneNumber } }
+        _set: { is_valid: $isValid }
+      ) {
+        affected_rows
+      }
+    }
+  `)
+  const [updateMemberMangerId] = useMutation(gql`
+    mutation UpdateMemberMangerId($memberId: String!) {
+      update_member(where: { id: { _eq: $memberId } }, _set: { manager_id: null }) {
+        affected_rows
+      }
+    }
+  `)
+
```

Comment:
updateMemberManagerId is ambiguous in the situation
1. change the naming, such as updateMemberManagerIdNull
2. passing parameters to SQL